packages/fast-components-react-msft/src/metatext/metatext.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from "react";
+import * as ReactDOM from "react-dom";
+import { get } from "lodash-es";
+import { Foundation, HandledProps, TypeLevel, TypographyTag } from "@microsoft/fast-components-react-base";
+import {
+    IMetatextHandledProps,
+    IMetatextManagedClasses,
+    IMetatextUnhandledProps
+} from "./metatext.props";
+import Typography from "../typography";
+import { IManagedClasses, IMetatextClassNameContract } from "@microsoft/fast-components-class-name-contracts-msft";
+
+/* tslint:disable-next-line */
+class Metatext extends Foundation<IMetatextHandledProps & IManagedClasses<IMetatextClassNameContract>, React.HTMLAttributes<HTMLSpanElement | HTMLParagraphElement>, {}> {
+    protected handledProps: HandledProps<IMetatextHandledProps & IManagedClasses<IMetatextClassNameContract>> = {
+        managedClasses: void 0,
+        tag: void 0
+    };
+
+    /**
+     * Stores HTML tag for use in render
+     */
+    private get tag(): TypographyTag {
```

Comment:
Any reason you're not relying on `defaultProps` here? You could then just pass `this.props.tag` directly to `Typography` and avoid the getter.

Reply:
Established pattern we have had throughout all other components when providing a tag. We're using both a getter and default props in most places. Happy to update, but would ask that we create an issue to align all of them.

Reply:
It's only a necessary pattern in certain circumstances:

The Typography component uses that pattern because we need to be *sure* that the tag passed as props is valid. Because of that, we need to use a function to return the prop. TypeScript does not allow us to use a function's return value as a JSX tag though; `<this.tag()>` is invalid. The workaround is to use a getter. You're just passing a prop *down* to `Typography` though, not creating an HTML element, so it's a different case.

My main point is that you're applying a default value while the props interface explicitly requires the value. If the component has a default, then it shouldn't require a value.

Reply:
Not saying it's *necessary*, saying it's how we've managed this with all other typography-derived components. I'll push an update.

Reply:
Props was also a mistake, but the above explanation is accurate. We should update elsewhere, but that can be done in another PR.